File: demo1/src/main/java/org/mosquito/demo/mapper/UserMapper.java
package org.mosquito.demo.mapper;

import org.mosquito.demo.model.User;
import org.apache.ibatis.annotations.Mapper;
import org.mosquito.framework.jdbc.mapper.BaseMapper;

/**
 * 数据访问接口
 *
 * @author jcchen
 * @Date 2018-5-18
 * 
 */
@Mapper
public interface UserMapper extends BaseMapper<User> {

}

File: demo1s/src/main/scala/org/mosquito/demo1s/mapper/UserMapper.scala
package org.mosquito.demo1s.mapper

import org.apache.ibatis.annotations.Mapper

import org.mosquito.framework.jdbc.mapper.BaseMapper
import org.mosquito.demo1s.model.User

/**
 * 数据访问接口
 *
 * @author jcchen
 * @Date 2018-5-28
 * 
 */
@Mapper
trait UserMapper extends BaseMapper[User] {

}
